Fix wrong validation on input field with default value

vincenteasy opened the pull request:
Avoid `must be defined` error on an input field having a default value defined in the schema:

```
type Query {
    structArg(i: InputType!): Boolean!
}
input InputType {
    name: String!
    defaultName: String! = "defaultFoo"
}
```
With variables:
```
{
  "var": {
     "name": "foo"
  }
}
```

**Before**:
Error received: `input: variable.defaultName must be defined`

**After**:
No more errors received, this is the expected behavior because a default value is defined in the schema.

vektah commented:
Thanks for the PR, it needs a test and a PR description before I can merge it.

vincenteasy commented:
@vektah 

> Thanks for the PR, it needs a test and a PR description before I can merge it.

Hi, Thanks for your review.

I've updated the description but I don't have any real test to add: I've just added a field with default value in the InputType defined in `vars.graphql`. 

Before the patch, there was an error, and no more errors afterwards.

Which test can I add in addition?

vincenteasy commented:
Hi @vektah,

Hope you are well. Could you please review this PR?

Thanks

skaji commented:
@lwc Could you create a new git tag (maybe v2.2.0?),
so that we can easily import this fix into our projects with go module?

skaji commented:
@lwc @vektah Sorry for bothering you again,
but could you create a new git tag?
